packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -77,9 +77,20 @@ def env_file(osparc_simcore_root_dir: Path, devel_environ: Dict[str, str]) -> Pa
         backup_path.unlink()
 
 
+@pytest.fixture(scope="module")
+def make_up_prod_environ():
+    if not "DOCKER_REGISTRY" in os.environ:
+        os.environ["DOCKER_REGISTRY"] = "local"
+    if not "DOCKER_IMAGE_TAG" in os.environ:
+        os.environ["DOCKER_IMAGE_TAG"] = "production"
+

```

Comment:
the variables need to be unset when the test fixture gets out of scope. My proposal:
```python
@pytest.fixture(scope="module")
def make_up_prod_environ(monkeypatch):
    if not "DOCKER_REGISTRY" in os.environ:
         monkeypatch.setenv("DOCKER_REGISTRY", "local")
  ...
```
here monkeypatch takes care of unsetting them once the fixture is done

More info and examples in https://docs.pytest.org/en/3.0.1/monkeypatch.html#example-setting-an-environment-variable-for-the-test-session